server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -720,7 +860,7 @@ public void setMessageShouldReturnCreatedMessageWhenSendingMessage() {
             // assert that message flags are all unset
             .body(ARGUMENTS + ".created", hasEntry(equalTo(messageCreationId), Matchers.allOf(
                 hasEntry(equalTo("isDraft"), equalTo(false)),
-                hasEntry(equalTo("isUnread"), equalTo(false)),
+                hasEntry(equalTo("isUnread"), equalTo(true)),
```

Comment:
Read the creation statement. Is "isUnread" property mentioned?

I'm not sure this behaviour should be altered.